src/Common/Update.php

```diff
@@ -119,12 +119,13 @@ public function orWhere($cond)
      *
      * @param string $col The column name.
      *
-     * @return $this
+     * @param array  $value
```

Comment:
@pavarnos I was talking about space `@param array  $value` vs `@param array $value` . I missed to understand you were talking about extra line in b/w @param. You can have an extra line in b/w @param or we can leave that for now :) .